src/core/distributions/math/GeometricDistribution.cpp

```diff
@@ -45,15 +45,15 @@ GeometricDistribution* GeometricDistribution::clone( void ) const
 double GeometricDistribution::computeLnProbability( void ) 
 {
     
-    double v = *value;
+    long v = *value;
     
```

Comment:
What is the rationale for switching to `long` here?  Should we use `long long` instead?  On windows, `long` is 32 bits and `long long` is 64 bits :grimacing:

Reply:
The reason is that the GeometricDistribution derives from TypedDistribution<long>, which has `long* value` defined. So this is simply the corrected version and doesn't hide what the internal value actually is.